library/maven

```diff
@@ -1,64 +1,64 @@
 Maintainers: Carlos Sanchez <carlos@apache.org> (@carlossg)
 GitRepo: https://github.com/carlossg/docker-maven.git
 
-Tags: 3.6.2-jdk-11-openj9, 3.6-jdk-11-openj9, 3-jdk-11-openj9
+Tags: 3.6.3-jdk-11-openj9, 3.6-jdk-11-openj9, 3-jdk-11-openj9
 Architectures: amd64, ppc64le, s390x
-GitCommit: 6f912331309f5ab9745a91ef5884f3a0b46dfc1d
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-11-openj9
 
-Tags: 3.6.2-jdk-11-slim, 3.6.2-slim, 3.6-jdk-11-slim, 3.6-slim, 3-jdk-11-slim, slim
+Tags: 3.6.3-jdk-11-slim, 3.6.3-slim, 3.6-jdk-11-slim, 3.6-slim, 3-jdk-11-slim, slim
 Architectures: amd64, arm64v8
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-11-slim
 
-Tags: 3.6.2-jdk-11, 3.6.2, 3.6-jdk-11, 3.6, 3-jdk-11, 3, latest
+Tags: 3.6.3-jdk-11, 3.6.3, 3.6-jdk-11, 3.6, 3-jdk-11, 3, latest
 Architectures: amd64, arm64v8
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-11
 
-Tags: 3.6.2-jdk-13, 3.6-jdk-13, 3-jdk-13
+Tags: 3.6.3-jdk-13, 3.6-jdk-13, 3-jdk-13
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-13
 Constraints: !aufs
 
-Tags: 3.6.2-jdk-14, 3.6-jdk-14, 3-jdk-14
+Tags: 3.6.3-jdk-14, 3.6-jdk-14, 3-jdk-14
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-14
 Constraints: !aufs
 
-Tags: 3.6.2-jdk-8-openj9, 3.6-jdk-8-openj9, 3-jdk-8-openj9
+Tags: 3.6.3-jdk-8-openj9, 3.6-jdk-8-openj9, 3-jdk-8-openj9
 Architectures: amd64, ppc64le, s390x
-GitCommit: 6f912331309f5ab9745a91ef5884f3a0b46dfc1d
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-8-openj9
 
-Tags: 3.6.2-jdk-8-slim, 3.6-jdk-8-slim, 3-jdk-8-slim
+Tags: 3.6.3-jdk-8-slim, 3.6-jdk-8-slim, 3-jdk-8-slim
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-8-slim
 
-Tags: 3.6.2-jdk-8, 3.6-jdk-8, 3-jdk-8
+Tags: 3.6.3-jdk-8, 3.6-jdk-8, 3-jdk-8
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-8
 
-Tags: 3.6.2-ibmjava-8-alpine, 3.6.2-ibmjava-alpine, 3.6-ibmjava-8-alpine, 3.6-ibmjava-alpine, 3-ibmjava-8-alpine, ibmjava-alpine
+Tags: 3.6.3-ibmjava-8-alpine, 3.6.3-ibmjava-alpine, 3.6-ibmjava-8-alpine, 3.6-ibmjava-alpine, 3-ibmjava-8-alpine, ibmjava-alpine
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
```

Comment:
This commit doesn't seem to exist on https://github.com/carlossg/docker-maven

Reply:
🤦‍♂  pushed now